src/device/esp/esp32.S

```diff
@@ -43,7 +43,7 @@ call_start_cpu0:
     rsync
 
     // Jump to the runtime start function written in Go.
-    j main
+    call4 main
```

Comment:
This was a bug in the existing ESP32 support: it didn't adhere to the ABI before. Using `call4` makes it adhere to the ABI.